### ISSUE TITLE
http://xx.xxx.xx.xx/messages 404 not found error

### ISSUE THREAD

**Issue by aungyin** [jahvi/chatty#7]
is it because of the database.php or .env configuration error?

my .env setting is as below, database.php config for DB is same as .env.
Do I need to setup my PUSHER_KEY? I have no experience of using Pusher.
--------
APP_ENV=production
APP_DEBUG=false
APP_KEY=BF3nmfzXJ2T6XU8EVkyHtULCtwnakK5k (Note, not a real key)

DB_HOST=xxxxxxxxxxxxx.cyxxxxxxxxxx.xx-central.amazonaws.com:3306
DB_DATABASE=MyAppsDatabaseName
DB_USERNAME=MyAWSRDSUserName
DB_PASSWORD=NotARealPassword

CACHE_DRIVER=file
SESSION_DRIVER=file
QUEUE_DRIVER=sync

MAIL_DRIVER=smtp
MAIL_HOST=mailtrap.io
MAIL_PORT=2525
MAIL_USERNAME=null
MAIL_PASSWORD=null
MAIL_ENCRYPTION=null

PUSHER_KEY=null
PUSHER_SECRET=null
PUSHER_APP_ID=null

![screen shot 2016-11-28 at 1 05 08](https://cloud.githubusercontent.com/assets/8940339/20649665/ad589dc6-b507-11e6-8a94-c56379c5ad47.png)

**Comment by jahvi** [comment]
Yes you need to create a free account on their site and copy the key, secret and app id, you don't really need to do anything else so no experience needed.

**Comment by aungyin** [comment]
I created the account of Pusher and configure key, secret and app id. but still got the same error.

I guess the problem may lied in database config. I am using mysql, and I found there is config for redis driver in config/broadcasting.php. How do you think?
